lib/bootleg/tasks.ex

```diff
@@ -90,6 +86,27 @@ defmodule Bootleg.Tasks do
     end)
   end
 
+  defp load_bootleg_tasks path do
+    path
+    |> File.ls!()
+    |> Enum.map(&(Path.join(path, &1)))
+    |> Enum.each(fn(x) ->
+      if File.dir?(x) do
+        load_bootleg_tasks(x)
+      else
+        load_bootleg_task(x)
+      end
+    end)
+  end
+
+  defp load_bootleg_task file do
+    Code.eval_string(
+      File.read!(file),
+      [],
+      %{__ENV__ | line: 1, file: file}
+    )
+  end
```

Comment:
We also have `Config.load` for evaluating elixir script. Both these seem to do the same thing. We might combine them here, or defer it for later, but just calling it out.

Reply:
This is one of the first refactor targets. We actually load in a few different spots across bootleg, (two just inside this file!) so i want to see if there is a a way we can combine at least a few of them. I know we have to load differently depending on whether it's an internal vs external task, but maybe there's still hope to unify everything under one